test/integration/RSR-integration.test.ts

```diff
@@ -4,41 +4,46 @@ import { expect } from 'chai'
 import hre, { ethers } from 'hardhat'
 
 import { ForkSpell } from '../../typechain/ForkSpell'
-import { MultiSigWalletWithDailyLimit } from '../../typechain/MultiSigWalletWithDailyLimit'
 import { ReserveRightsToken } from '../../typechain/ReserveRightsToken'
 import { RSR } from '../../typechain/RSR'
 import { SiphonSpell } from '../../typechain/SiphonSpell'
-import { SlowWallet } from '../../typechain/SlowWallet'
 import { UPGRADE_SIPHONS } from './../../scripts/deployment/siphon_config'
 import { impersonate } from './utils/accounts'
-import { ONE, ZERO } from '../../common/numbers'
+import { ONE, ZERO, bn } from '../../common/numbers'
 import { WEIGHT_ONE, ZERO_ADDRESS } from '../common'
 
+// TODO: Add more siphon test cases when the contract is deployed
+// For the mainnet fork, only test the first 5 addresses with balances using different weight values
+const mockWeights = [bn('0'), bn('0.5e18'), WEIGHT_ONE, bn('0.2e18'), bn('0.75e18')]
+const mockSiphons = UPGRADE_SIPHONS.slice(0, 5).map((siphon, index) => ({
+  ...siphon,
+  weight: mockWeights[index],
+}))
+
 // Relevant addresses (Mainnet)
 const RSR_PREVIOUS_ADDRESS = '0x8762db106b2c2a0bccb3a80d1ed41273552616e8'
 const RSR_PAUSER_ADDRESS = '0xBb20467EcccB3F60F8dbEca09a61879893e44069'
-const SLOW_WALLET = '0x4903DC97816f99410E8dfFF51149fA4C3CdaD1b8'
-const MULTISIG_WALLET = '0xb268c230720D16C69a61CBeE24731E3b2a3330A1'
 const HOLDER_ADDRESS = '0x72A53cDBBcc1b9efa39c834A540550e23463AAcB'
+// const SLOW_WALLET = '0x4903DC97816f99410E8dfFF51149fA4C3CdaD1b8'
```

Comment:
Iwould remove all reference to these variables in the code if we are not using them we can add them later but it would be better to remove all that commented lines I think for now.